### DISCUSSION TITLE
Double space for period

### DISCUSSION THREAD

**Discussion by marlonjames71** [notaapp/talk#281] [Feature Requests]
First off, I'm definitely a fan of Nota. It has one of, if not, the best editor experiences I've ever used in a note-taking tool — finally, a markdown note tool where the editor feels close to the power features you get in an IDE like VS Code or Xcode which is what I use on a daily basis.

As a Mac user, the one thing I get a little irritated by is no option for a double space substitution for a period. This is virtually everywhere on macOS and I'm really used to it by now. Would love to see that come to Nota. I'm sure there are folks that don't want this, but I think an option to turn it on/off would be great!

Thanks for hearing me out!

**Comment by astoilkov** [comment]
I agree. We want it ourselves. I tried to do it once but wasn't able to. I will look into this once again.

Thanks!

> First off, I'm definitely a fan of Nota. It has one of, if not, the best editor experiences I've ever used in a note-taking tool — finally, a markdown note tool where the editor feels close to the power features you get in an IDE like VS Code or Xcode which is what I use on a daily basis.

❤️ ❤️ ❤️

**Reply by marlonjames71** [reply]
Ahh okay. That would be great if you were able to figure it out at some point. Thanks for the quick reply!

**Reply by marlonjames71** [reply]
Hey @astoilkov ... I see the label has been changed to `resolved in future release`. Am I right to assume that Nota is an Electron app? If so, I asked about this in Electron's Discord and someone responded saying that it should be an app-level implementation. They also shared that the following code snippet loaded into Electron 21 will add a period if you double-tap the space bar:

```
<blockquote contenteditable="true">
    <p>Edit this content to add your own quote</p>
</blockquote>
```

I was curious and asked and thought maybe this could help.

**Reply by astoilkov** [reply]
Yeah, it's done. The next release (in 1-2 weeks) will include this.

Yes, Nota is an Electron app. Yep, I know about this solution, however, we can't use `contenteditable` so I did find another solution. Thanks for helping, that's awesome!

**Reply by marlonjames71** [reply]
Oh nice. Good news!

**Comment by marlonjames71** [comment]
Thanks a ton for adding this! **It works flawlessly**!! Cheers mates! 😃
